Geometry/HGCalCommonData/src/HGCalCellOffset.cc

```diff
@@ -57,23 +62,31 @@ HGCalCellOffset::HGCalCellOffset(
                                                       -yMag,
                                                       (-0.5 * yMag + sqrt3By2_ * xMag),
                                                       (0.5 * yMag - sqrt3By2_ * xMag)}));
-      } else if (j == 1) {
-        double totalArea = (5.0 * sqrt3_ / 4.0) * std::pow(cellY_[k], 2);
-        double cutArea = cellY_[k] * sqrt3_ * guardRingOffset_;
+      } else if (j == 1) {                                                 // Offset for truncated cells
+        double totalArea = (5.0 * sqrt3_ / 4.0) * std::pow(cellY_[k], 2);  // Area of cell without any dead zone
+        double cutArea =
+            cellY_[k] * sqrt3_ * guardRingOffset_;  // Area of inactive region form guardring and other effects
         cellArea[k][j] = totalArea - cutArea;
         double offMag = (((-2.0 / 15.0) * totalArea * cellY_[k]) - ((cellY_[k] - (0.5 * guardRingOffset_)) * cutArea)) /
-                        (cellArea[k][j]);
+                        (cellArea[k][j]);  // Magnitude of offset
+        // (x, y) coordinates of offset for 6 sides of wafer starting from bottom left edge in clockwise direction
+        // offset_x = -Offset_magnitude * sin(30 + 60*i) i in (0-6)
+        // offset_y = -Offset_magnitude * cos(30 + 60*i) i in (0-6)
         tempOffsetX.emplace_back(
             std::vector<double>({-0.5 * offMag, -offMag, -0.5 * offMag, 0.5 * offMag, offMag, 0.5 * offMag}));
         tempOffsetY.emplace_back(std::vector<double>(
             {-sqrt3By2_ * offMag, 0.0, sqrt3By2_ * offMag, sqrt3By2_ * offMag, 0.0, -sqrt3By2_ * offMag}));
-      } else if (j == 2) {
-        double totalArea = (7.0 * sqrt3_ / 4.0) * std::pow(cellY_[k], 2);
-        double cutArea = cellY_[k] * sqrt3_ * guardRingOffset_;
+      } else if (j == 2) {                                                 //Offset for truncated cells
```

Comment:
The comment seems wrong, right? It should read `extended`, according to the previous comment.
What I had in mind was, in fact, to use some `enum` with expressive names, rather than commenting in the code what `j=0,1,2` means.
Aren't those concepts defined as `enums` in the `HGCAL` geometry?

Reply:
I have changed the format to incorporate the definitions form HGCalCell.h